mqtt-streaming/src/main/mima-filters/2.0.0-M3.backwards.excludes/ClassicActorSystemProvider

```diff
@@ -0,0 +1,11 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.alpakka.mqtt.streaming.javadsl.ActorMqttServerSession.this")
```

Comment:
Maybe add a comment why we don't care. Do we just not care about binary compat here?

Reply:
We are going for a major release and this connector is Akka 2.6 only now.